### DISCUSSION TITLE
Prevent background scrolling

### DISCUSSION THREAD

**Discussion by sebastian-vass** [FRSOURCE/cypress-plugin-visual-regression-diff#306] [Q&A]
### Application behaviour

We will insert a button on the client's page that, when clicked, will open our application. The application runs in the Shadow-DOM and in the open state a we create a new DOM in which our application is displayed. Basically, the entire original DOM is overlaid and there is no way to scroll on it. So the scrollable part is just our application's in the shadow DOM.

### Issue
When I set fullpage scrolling, it scrolls the original DOM and ignores the Shadow DOM above it.

### Question
Is there any way to determine which DOM I want to screenshot?

**Comment by FRSgit** [comment]
Hey!

It seems like a Cypress fullPage feature issue. Unfortunately - they don't have any official guide on how to overcome this issue (and quite long list [of issues that reference it](https://github.com/cypress-io/cypress/issues?q=is%3Aissue+fullPage+fixed+)).

So you need to do some manual work to fix this. My idea would be to read the scroll height of your shadowDOM application and then apply this height on the parent element which hold shadowDOM app. Effectively this should make a "shadowDOM scroll" hide (as it's fully expanded) and in turn - show the browser's scroll. And that one should be handled by Cypress correctly.

But as I said in the beginning - it's not an issue with this plugin, but Cypress, so please raise an issue on[ their github board.](https://github.com/cypress-io/cypress/issues)

Regards!
